tools/aa/commands/InitCommand.java

```diff
@@ -96,11 +98,34 @@ public boolean run(String[] args) {
     return true;
   }
 
+  private boolean processArgs(String[] args) {
+    if (args.length == 0 || args.length == 1) {
```

Comment:
Let's check if `aa` is set up before this message, and then print the relevant part of the message.

Reply:
Done it. I also added a `switch` instead of `if else` constructions. It should be more understandable.